CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributor Guideline
+
+
+## Commit
+
+> One golden rule: follow [the commit format][commit-format]
+
+We use [standard-version][standard-version] to automatically generate [CHANGELOG.md][CHANGELOG.md] from the `git commit` history. Please **do** follow [the commit format][commit-format] before commiting.
+
+
+## PR
+
+Simple rules:
+
+* Do rebase before merging.
```

Comment:
Yes, we don't merge `master` **into** `branch`. We only do rebase the branch **onto** latest `master`.